### PR TITLE
Pikaday package override.

### PR DESCRIPTION
**Pull request by akash1810**
Loading css and js together.

### CODE REVIEW

File: package-overrides/github/dbushell/Pikaday/Pikaday@1.3.1.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	"files": [
```
Comment (guybedford) [inline package-overrides/github/dbushell/Pikaday/Pikaday@1.3.1.json]: Rather leave this out.

File: package-overrides/github/dbushell/Pikaday/Pikaday@1.3.1.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		"pikaday.js",
+		"css/pikaday.css"
+	],
+	"dependencies": {
+		"css": "*"
+	},
+	"shim": {
+		"pikaday.js": {
+			"deps": [
+				"pikaday.js",
```
Comment (guybedford) [inline package-overrides/github/dbushell/Pikaday/Pikaday@1.3.1.json]: Why does it need to depend on itself?

File: package-overrides/github/dbushell/Pikaday/Pikaday@1.3.1.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+				"css/pikaday.css!"
```
Comment (guybedford) [inline package-overrides/github/dbushell/Pikaday/Pikaday@1.3.1.json]: This should probably be `'./css/pikaday.css!'`

**Comment by akash1810** [comment]
@guybedford thanks for your feedback.

**Pull request merged**